modules/database/src/std/rec/mbboDirectRecord.c

```diff
@@ -329,9 +298,24 @@ static void monitor(mbboDirectRecord *prec)
     if (prec->mlst != prec->val) {
         events |= DBE_VALUE | DBE_LOG;
         prec->mlst = prec->val;
+        db_post_events(prec, &prec->mlst, events);
```

Comment:
I don't think we should be posting events on the MLST field (no other records do), and having said that I'm not convinced that doing so on the OBIT field makes much sense either.

Reply:
Again, debugging which can be removed.  Being able to use an OPI is much more convenient than the alternatives.  Lots of `printf()` or parsing frequent `dbgf()`.